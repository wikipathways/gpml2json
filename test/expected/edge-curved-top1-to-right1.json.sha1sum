05db9690d3426aa62c642892b5c5715889b4a4b3 ?-
