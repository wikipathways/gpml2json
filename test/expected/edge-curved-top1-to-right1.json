{"pathway":{"@context":["https://cdn.rawgit.com/wikipathways/WpVocabularies/7a46a05/contexts/pvjs.jsonld"],"backgroundColor":"white","borderWidth":0,"color":"black","contains":["d6ee8","af9e0","a0723"],"drawAs":"rect","gpmlElementName":"Pathway","height":825.09187756604,"kaavioType":"Group","name":"test case","padding":5,"type":["Pathway"],"width":1047.7865523249097,"x":0,"y":0,"zIndex":0,"fontSize":12,"fontWeight":"bold","textAlign":"left","verticalAlign":"top","dataSourceVersion":"20130813","standardName":"test case","displayName":"test case","textContent":"Name: test case","isSimilarTo":"http://wikipathways.org/index.php/Special:SearchPathways?query=test%20case&doSearch=1","id":"hash://sha256/482ab8c0e2f133e3a5b9eaf8d16b34e237bc587d126ea8a5df71c8506f730b4e"},"entityMap":{"af9e0":{"id":"af9e0","textContent":"GeneProduct","wpType":"GeneProduct","x":196.7289428223533,"y":114.44624526539923,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":32954,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"a0723":{"id":"a0723","textContent":"0006","wpType":"GeneProduct","x":14.993106861058784,"y":84.27400896748527,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":32955,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"d6ee8":{"id":"d6ee8","drawAs":"CurvedLine","borderWidth":1,"zIndex":32953,"points":[{"isAttachedTo":"a0723","attachmentDisplay":{"position":[0.5,0],"offset":[0,0]},"x":54.993106861058784,"y":84.27400896748527,"orientation":[0,-1]},{"y":64.27400896748527,"x":115.86102484170604},{"x":176.7289428223533,"y":94.36012711644224},{"isAttachedTo":"af9e0","attachmentDisplay":{"position":[0,0.5],"offset":[0,0]},"x":196.7289428223533,"y":124.44624526539923,"orientation":[1,0]}],"color":"#000000","type":["Interaction","Edge"],"kaavioType":"Edge","gpmlElementName":"Interaction","markerStart":"TBar","markerEnd":"Arrow","isAttachedTo":["a0723","af9e0"]}}}
